src/components/NcSelect/NcSelect.vue

```diff
@@ -1010,24 +996,21 @@ export default {
 		},
 
 		propsToForward() {
-			const {
-				// Props handled by this component
-				inputClass,
-				noWrap,
-				placement,
-				userSelect,
-				// Props to forward
-				...initialPropsToForward
-			} = this.$props
-
+			const vueSelectKeys = [
+				...Object.keys(VueSelect.props),
+				...VueSelect.mixins.flatMap(mixin => Object.keys(mixin.props ?? {})),
```

Comment:
Off-top: mixins with props make me cry 🥲

Reply:
Exactly what I was doing while writing this 🙃